decidim-initiatives/app/commands/decidim/initiatives/vote_initiative.rb

```diff
@@ -92,6 +93,21 @@ def notify_percentage_change(before, after)
           }
         )
       end
+
+      def notify_support_threshold_reached(percentage)
+        return unless percentage >= 100
+
+        Decidim::EventsManager.publish(
+          event: "decidim.events.initiatives.support_threshold_reached",
+          event_class: Decidim::Initiatives::Admin::SupportThresholdReachedEvent,
+          resource: @initiative,
+          followers: organization_admins
+        )
+      end
+
+      def organization_admins
```

Comment:
As I saw in last paragraph of this @virgile-dev comment, he asked to send a notification to the admin when threshold was reached:
![image](https://user-images.githubusercontent.com/9702463/82569287-51e06100-9b80-11ea-90c7-f0167362c9db.png)
but as I saw there:
![image](https://user-images.githubusercontent.com/9702463/82569406-7dfbe200-9b80-11ea-8de9-4849df4da893.png)
he makes a changelog PR description's change suggestion as if the notification had to be sent to all them

As code shows the notification only will be sent to organization's admins, could you check/clarify this please @armandfardeau @virgile-dev ?

Thanks in advance!

Reply:
@armandfardeau I was under the impressions we'd notify the author, the promoting committee and the followers. Can you make the necessary changes please ?

Reply:
For sure, sorry for the mess.

As far as I understood, we already send a notification to authors, followers, and committee members. 
The notification is included is `notify_percentage_change`.

In order to customize the notifications, a new method has been added `notify_support_threshold_reached` which notifies only admins.

Let me know if issues are still hanging around

Reply:
So method `notify_percentage_change` is already sending a notification to authors, followers and commitee members for percentage changes between 25, 50, 75, 100 ?
It looks like as author wouldn't be notified and committee members would be only if they already are initiative's followers, is this right?

![image](https://user-images.githubusercontent.com/9702463/82656467-1d25e580-9c24-11ea-9650-5bf97e286085.png)

The new `notify_support_threshold_reached` method only will send a notification to admin when 100% is reached.

Could you confirm that this is how it should be please?
Thanks in advance

Reply:
@Leusev Yes, this way we can customize the admin notifications

@virgile-dev Mind to double-check?

Reply:
Yes it's perfect to have a different one for admins for customization purposes. I approve 👍